frontend/awx/resources/groups/GroupEdit.tsx

```diff
@@ -0,0 +1,61 @@
+import { useTranslation } from 'react-i18next';
+import { AwxPageForm } from '../../common/AwxPageForm';
+import { InventoryGroup, InventoryGroupCreate } from '../../interfaces/InventoryGroup';
+import {
+  PageFormDataEditor,
+  PageFormSubmitHandler,
+  PageFormTextInput,
+  usePageNavigate,
+} from '../../../../framework';
+import { PageFormSection } from '../../../../framework/PageForm/Utils/PageFormSection';
+import { useNavigate } from 'react-router-dom';
+import { awxAPI } from '../../common/api/awx-utils';
+import { AwxRoute } from '../../main/AwxRoutes';
+import { usePutRequest } from '../../../common/crud/usePutRequest';
+
+export function GroupEdit(props: { group: InventoryGroup }) {
+  const { t } = useTranslation();
+  const group = props.group;
+  const navigate = useNavigate();
+  const putRequest = usePutRequest<InventoryGroupCreate, InventoryGroup>();
+  const pageNavigate = usePageNavigate();
+
+  const onSubmit: PageFormSubmitHandler<InventoryGroupCreate> = async (groupInput) => {
+    const { name, description, variables } = groupInput;
+    const editGroup: InventoryGroupCreate = {
+      name,
+      description,
+      variables,
+      inventory: group.inventory,
+    };
+    const editedGroup = await putRequest(awxAPI`/groups/${group.id.toString()}`, editGroup);
```

Comment:
Looking at the other forms in our project, we should follow the pattern of using `patchRequest` instead of `putRequest` here.

Reply:
The request URL is missing a trailing `/` in the url.